### PR TITLE
Nixoflife

### PR DESCRIPTION
**Pull request by AbyssalRemark**
A nix version for game of life thus far.

Still has all the rust code in lib.rs, will change soon sense this is software construction. Not warm yet. But, it should work every single time.

### CODE REVIEW

**Comment by poperigby** [comment]
You can also declare the Rust dependencies with https://github.com/cargo2nix/cargo2nix

File: sc-24sp/assignments/abyssalremark/flake.nix.save
[the remark this answers is not in the document]
Comment (AbyssalRemark) [reply sc-24sp/assignments/abyssalremark/flake.nix.save:1]: eh, I mean I want it for my own records. its not strictly *nessissary* no. I actually deleted it once and then went. eh. whats the harm

yea lets clip it out

**Comment by AbyssalRemark** [comment]
> You can also declare the Rust dependencies with https://github.com/cargo2nix/cargo2nix

Going to need to look into that. Either way this should work for now.

**Comment by poperigby** [comment]
Uhhh you deleted your flake.nix instead of your flake.nix.save. Was that intentional?

**Comment by AbyssalRemark** [comment]
> Uhhh you deleted your flake.nix instead of your flake.nix.save. Was that intentional?

I should not do things at 1 in the morning. *face palm*

**Pull request merged**